tests/integ/admin_server/logendpoint_test.go

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+//
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+
+//go:build integration
+// +build integration
+
+package test
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"flag"
+	"fmt"
+	"net/http"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/linuxboot/contest/cmds/admin_server/storage"
+	mongoAdapter "github.com/linuxboot/contest/cmds/admin_server/storage/mongo"
+	"github.com/stretchr/testify/require"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+	"go.mongodb.org/mongo-driver/mongo/options"
+)
+
+var (
+	flagAdminEndpoint = flag.String("adminServer", "http://adminserver:8000/log", "admin server log push endpoint")
+	flagMongoEndpoint = flag.String("mongoDBURI", "mongodb://mongostorage:27017", "mongodb URI")
+)
+
+func setupClearDB(uri string) (*mongo.Client, error) {
```

Comment:
nit
```suggestion
func setupCleanDB(uri string) (*mongo.Client, error) {
```

or empty (clear is more like "transparent")